### ISSUE TITLE
"Preset OS shell" asks for password

### ISSUE THREAD

**Issue by theganyo** [TheNewNormal/kube-solo-osx#76]
When I run the "Preset OS shell" option, I get asked for a password. Should it do that? If so, what password does it expect?

```
$ /Applications/Kube-Solo.app/Contents/Resources/os_shell.command; exit;

etcdctl ls /:
core@192.168.64.6's password:
core@192.168.64.6's password:
core@192.168.64.6's password:
```

Thanks!

**Comment by rimusz** [comment]
@theganyo did your VM start normally? 
You should not be asked for the password.

**Comment by theganyo** [comment]
Yes, it seemed to:

```
Starting VM ...
> booting k8solo-01
[corectl] stable/1010.5.0 already available on your system
[corectl] '/Users/sganyo' was already available to VMs via NFS
[corectl] started 'k8solo-01' in background with IP 192.168.64.6 and PID 26243

Waiting for VM to be ready...
\
fleetctl list-machines:
MACHINE     IP      METADATA
68b74dfb... 192.168.64.6    -
6b335431... 192.168.64.6    -

Waiting for Kubernetes cluster to be ready. This can take a few minutes...
-
kubernetes nodes list:
NAME           STATUS    AGE
192.168.64.6   Ready     20h
```

**Comment by theganyo** [comment]
Also, I attempted to launch the "Kubernetes Dashboard" and received this error:

```
Internal Server Error (500)

the server has asked for the client to provide credentials (get replicationControllers)
```

**Comment by rimusz** [comment]
@theganyo can you do the fresh VM install, something must be went wrong there

**Comment by theganyo** [comment]
Ok. Here's what I did:
- Destroy VM
- rm -rf ~/kube-solo
- Initial Setup (Stable)
  (The only error I got here was: "[ERROR] Remote kube-charts already exists, and is pointed to https://github.com/TheNewNormal/kube-charts")
- Up

All of that seemed to go just fine.
Also, Kubernetes dashboard seems to work now.

The original issue with "Preset OS shell" remains, however. 

Is there anything else I could have cleaned up?
